packages/next/build/babel/plugins/next-ssg-transform.ts

```diff
@@ -1,455 +0,0 @@
-import {
```

Comment:
This file will have to be preserved to ensure backwards compat

Reply:
I restored it.